src/webgpu/api/validation/image_copy/texture_related.spec.ts

```diff
@@ -175,6 +175,73 @@ g.test('mip_level')
     );
   });
 
+g.test('format')
+  .desc(`Test that it must be a full copy if the texture's format is depth/stencil format`)
+  .params(u =>
+    u //
+      .combine('method', kImageCopyTypes)
+      .combineWithParams([
+        { depthOrArrayLayers: 1, dimension: '2d' },
+        { depthOrArrayLayers: 3, dimension: '2d' },
+        { depthOrArrayLayers: 32, dimension: '3d' },
+      ] as const)
+      .combine('format', kSizedTextureFormats)
+      .filter(({ dimension, format }) => textureDimensionAndFormatCompatible(dimension, format))
+      .filter(formatCopyableWithMethod)
```

Comment:
BTW, this test actually can't test what I wanted to test, because copySrc/copyDst is marked as false in capability_info.js. As a result, depth/stencil formats won't be tested in this test. 

We discussed the issue about copySrc/copyDst of capability_info.js at https://github.com/gpuweb/cts/pull/481. But we didn't address that issue because it is not finalized in WebGPU spec. Now that the spec says that we can do full copy for depth/stencil format, I think it's time to change copySrc/copyDst in capability_info.js. WDYT?

Reply:
I agree we should do it. I am not exactly sure how we should represent it in the table. Maybe one sub-structure for each aspect? `{ color: {...} | undefined, depth: {...} | undefined, stencil: {...} | undefined }`
but it would result in some churn in the rest of the project.

Another option, we could use the existing fields just for color, and add sub-structures (or just new fields) for depth/stencil?

We'd probably also need some helper that takes a GPUTextureAspect and fetches the appropriate aspect sub-structure for you.

Reply:
Hmm, I found that we have already had function depthStencilBufferTextureCopySupported and kDepthStencilFormatCapabilityInBufferTextureCopy. We can use that function.

Reply:
I added tests for SizedDepthStencilFormats based on existing information/functions in capability_info.ts. I also added a couple more functions. UnsizedDepthStencilFormats will be the next step, whose BytesPerBlock should be set per aspect but it is set per format in capability_info.ts.